From 59cf3c7dd08fffa33559a7df76a81350580f1af1 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Tue, 19 Sep 2023 16:50:46 +0200
Subject: [PATCH 0525/1141] arm64: defconfig: Enable I3C for STM32MP25

Enable the I3C subsystem and STM32 I3C controller driver to be used as
I3C master.

Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Change-Id: I0d258120af51f69e521e49ffc00750ec424f2b8f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/328458
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -30,6 +30,7 @@ CONFIG_DYNAMIC_DEBUG=y
 CONFIG_HDMI=y
 CONFIG_I2C_RPMSG=m
 CONFIG_I2C_STM32F7=y
+CONFIG_I3C=y
 CONFIG_IIO_STM32_LPTIMER_TRIGGER=m
 CONFIG_IIO_STM32_TIMER_TRIGGER=m
 CONFIG_INPUT_TOUCHSCREEN=y
@@ -81,6 +82,7 @@ CONFIG_STM32_ADC_CORE=m
 CONFIG_STM32_DMA3=y
 CONFIG_STM32_FMC2_EBI=y
 CONFIG_STM32_HYPERBUS=y
+CONFIG_STM32_I3C_MASTER=m
 CONFIG_STM32_IPCC=y
 CONFIG_STM32_LPTIMER_CNT=m
 CONFIG_STM32_MDF_ADC=m
